### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tykhan**
- [DEMO LINK](https://tykhan.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://tykhan.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  padding-left: 62px;
+}
+
+.smaller-bar {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.big-dandruff {
```
Comment (irenhh) [inline src/style.css:35]: big what? :)

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+.small-dandruff {
+  left: 13px;
+  top: 15px;
+  height: 11px;
+  width: 11px;
+}
+
+.icon {
+  position: absolute;
+}
```
Comment (irenhh) [inline src/style.css:51]: position is a property, which is good to have separately for each element, no need to move it to general class

File: src/style.css (hunk 1 above)
Comment (tykhan) [reply src/style.css:35]: ahah, лупа in english, for native speakers will be clear:)

Comment (tykhan) [reply src/style.css:35]: but anyway, I'll change it)

**Pull request closed without merging**